docs/content/content/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Content
+description: Basic principles to create clear and consistent content in the Export Service. Learn how to express, structure and style content that meets users’ needs.
+---
+
+---
```

Comment:
Seems weird to start the page off with a Divider... I wonder if it would be better to use our `Divider` component instead (in the template file)?